app/home/page.tsx

```diff
@@ -0,0 +1,82 @@
+import css from "classnames";
+import {
+  Button,
+  Header,
+  Heading,
+  Layout,
+  DailyCard,
+  CategoryCard,
+  Icon,
+  Pill,
+} from "../components";
+import styles from "./home.module.css";
+import { Colors } from "../constants/colors";
+
+type Props = {};
+
+const Home = ({}: Props): JSX.Element => {
+  const userName = "Erlich Bachman";
+
+  return (
+    <Layout>
+      <Header page="home" />
+
+      <Heading variant="v100">Hello</Heading>
+      <Heading variant="v100">{userName}</Heading>
+
+      <div className={styles["button-wrapper"]}>
+        <Button title="Overview" size="small" theme="primary" />
+        <Button title="Productivity" size="small" theme="clear" />
+      </div>
+
+      <div style={{ height: "140px" }}>
+        <DailyCard className={css(styles.card, styles["first-card"])} />
+        <DailyCard className={css(styles.card, styles["second-card"])} />
+        <DailyCard className={css(styles.card, styles["third-card"])} />
+      </div>
+
+      <Heading variant="v500">Categories</Heading>
+
+      <div className={styles["category-container"]}>
```

Comment:
I think we can have a list with the categories data and intereate it to create this components.

Reply:
done!